pkg/cmd/roachtest/tests/mixed_version_cdc.go

```diff
@@ -135,6 +135,7 @@ func (cmvt *cdcMixedVersionTester) Cleanup() {
 	if cmvt.cleanup != nil {
 		cmvt.cleanup()
 	}
+	cmvt.monitor.Wait()
```

Comment:
From my memory of this test, I don't think this is going to work. One of the goroutines just runs a workload indefinitely and relies on context cancelation to terminate. The context is only canceled by roachtest when the `Run` function returns, so this `Wait` call will block "forever" (until test timeout).

I'm less concerned about this kind of leak (as the function will exit shortly after the test finishes), but you can create an issue and I can update with the proper context dance.

Reply:
Thanks!